Change path to conclave repo

snedamle opened the pull request.

snedamle committed:
update conclave repo path, add links to all samples on root README.md

snedamle committed:
update conclave repo path, add links to all samples on root README.md

snedamle committed:
update conclave repo path, add links to all samples on root README.md

gendal commented:
Hi Sneha - nice to see the additional polish on the samples!

Re the repo location, did the global gradle settings 'trick' not work?

snedamle commented on README.md:
```diff
@@ -10,6 +10,46 @@ developers to get started with the conclave platform and understand different fe
+
+If you wish to download the conclave sdk repo to another location, please update this conclaveRepo property in 
+gradle.properties for all the samples.
+
```
@gendal I have mentioned in the Readme to download the SDK and save it in the conclave-samples/conclave-sdk/{version-number}/ 
and accordingly have updated the location in gradle to point to ../conclave-sdk/1.1/repo. 

So now all the samples will work if they have the sdk at this location conclave-samples/conclave-sdk/{version-number}/ 
This looked to be a quick fix. 

But let me know your thoughts on this.

gendal replied:
Hi Sneha - I was referring to the discussion in the comments of this Jira: https://r3-cev.atlassian.net/browse/CON-574. @rusheb  outlined a global way to set the repo directory that _eliminated_ the need for users of the samples to have to edit multiple files.

snedamle replied:
@gendal ok got you, will try this out.

snedamle committed:
default location for sdk will be ~/conclave-sdk/{version}. if the user wishes to add a custom location he can update the conclaveRepo property in gradle.properties.
This requires a change in container-gradle script. I have added the these to a sample container-gradle in psi project.

snedamle committed:
update README to reflect the changes
